.storybook/style.scss

```diff
@@ -0,0 +1,6 @@
+$formation-asset-path: '~@department-of-veterans-affairs/formation/assets';
+
+@import '@department-of-veterans-affairs/formation/sass/base/_b-mixins.scss';
+@import '@department-of-veterans-affairs/formation/sass/base/_b-variables.scss';
+@import '@department-of-veterans-affairs/formation/sass/base/_b-breakpoints.scss';
+@import '@department-of-veterans-affairs/formation/sass/site/_m-crisis-line.scss';
```

Comment:
All of these are necessary in order to get the `crisis-line` module to load.

Reply:
:scream: